src/main/java/com/volmit/iris/util/decree/virtual/VirtualDecreeCommand.java

```diff
@@ -462,7 +462,7 @@ private boolean invokeNode(VolmitSender sender, KMap<String, Object> map) {
 
             try
             {
-                if(value == null && !i.getParam().defaultValue().trim().isEmpty())
+                if(value == null && i.hasDefault())
```

Comment:
see above, no.

Reply:
Unsure why the hasDefault call here is wrong
hasDefault now does: `return !param.defaultValue().trim().isEmpty();`